packages/api/tests/endpoints/test-rules.js

```diff
@@ -1626,6 +1607,29 @@ test('DELETE deletes a rule', async (t) => {
   );
 });
 
+test('DELETE deletes only a Rule\'s resources if onlyResources queryparam is true', async (t) => {
+  const {
+    originalPgRecord,
+  } = await createRuleTestRecords(
+    t.context,
+    {
+      workflow,
+    }
+  );
+  t.true(await t.context.rulePgModel.exists(t.context.testKnex, { name: originalPgRecord.name }));
+
+  const response = await request(app)
+    .delete(`/rules/${originalPgRecord.name}?onlyResources=true`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  const { message } = response.body;
+
+  t.true(await t.context.rulePgModel.exists(t.context.testKnex, { name: originalPgRecord.name }));
+  t.is(message, 'Record resources (e.g. CloudWatch Events, Kinesis Event Sources) deleted');
```

Comment:
This doesn't actually test that the rule's resources are deleted. My thinking is that this is an endpoint test so verifying that we hit the correct logic block seems appropriate and `deleteRuleResources` is unit tested elsewhere.